### PR TITLE
add warning about what disaster recovery does to a running cluster

### PR DESCRIPTION
**Pull request by bergerhoffer**
Replaces #34842.

Preview: 
* https://deploy-preview-35439--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/about-disaster-recovery.html
* https://deploy-preview-35439--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state-about_dr-restoring-cluster-state

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@deads2k for review

Preview:

* https://deploy-preview-35439--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/about-disaster-recovery.html
* https://deploy-preview-35439--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state-about_dr-restoring-cluster-state

**Comment by deads2k** [comment]
lgtm

**Comment by bergerhoffer** [comment]
@geliu2016 Can you please QE review?

Preview:

* https://deploy-preview-35439--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/about-disaster-recovery.html
* https://deploy-preview-35439--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state-about_dr-restoring-cluster-state

File: modules/dr-restoring-cluster-state-about.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//
+// * disaster_recovery/scenario-2-restoring-cluster-state.adoc
+
+[id="dr-scenario-2-restoring-cluster-state-about_{context}"]
+= About restoring cluster state
+
+You can use an etcd backup to restore your cluster to a previous state. This can be used to recover from the following situations:
+
+* The cluster has lost the majority of control plane hosts, leading to etcd quorum loss and the cluster going offline.
+* An administrator has deleted something critical and must restore to recover.
```
Comment (hexfusion) [inline modules/dr-restoring-cluster-state-about.adoc:11]: just minor tweak clarifying that the cluster is assumed unrecoverable without this action.

`must restore to recover the cluster.`

File: modules/dr-restoring-cluster-state-about.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/scenario-2-restoring-cluster-state.adoc
+
+[id="dr-scenario-2-restoring-cluster-state-about_{context}"]
+= About restoring cluster state
+
+You can use an etcd backup to restore your cluster to a previous state. This can be used to recover from the following situations:
+
+* The cluster has lost the majority of control plane hosts, leading to etcd quorum loss and the cluster going offline.
```
Comment (hexfusion) [inline modules/dr-restoring-cluster-state-about.adoc:10]: clarify that this is quorum loss scenario 

`The cluster has lost the majority of control plane hosts (quorum loss)`

**Review by hexfusion — COMMENTED** [review]
couple notes otherwise LGTM

**Comment by hexfusion** [comment]
LGTM

**Comment by stevsmit** [comment]
LGTM!

**Comment by geliu2016** [comment]
LGTM, thanks

**Pull request merged**